docker-compose.yml

```diff
@@ -7,31 +7,31 @@ services:
       - ./chains/privnet-docker-one:/chains/privnet
     ports:
       -  20334:20334
-    command: "node --config-path /config --privnet"
+    command: "node --privnet"
```

Comment:
I'm not sure about this one. We have 
```
    volumes:
      - ./config/protocol.privnet.docker.one.yml:/config/protocol.privnet.yml
```
In the top of `docker-compose.yml`, so I guess this path (as strange as it is) is correct for this particular scenario.

What I would probably change here is port numbers to match standard C# privnet setup.

Reply:
> What I would probably change here is port numbers to match standard C# privnet setup.

But that would require changing configs also.